source/SpinalHDL/Data types/Int.rst

```diff
@@ -392,12 +392,21 @@ Misc
    * - x.lsb
      - Return the least significant bit
      - Bool
-   * - x.range
-     - Return the range (x.high downto 0)
-     - Range
    * - x.high
      - Return the upper bound of the type x
      - Int
+   * - x.bitsRange
+     - Return the range (x.high downto 0)
+     - Range
+   * - x.minValue
+     - Lowest value that x can take on (e.g. 0 for UInt)
+     - BigInt
+   * - x.maxValue
+     - Highest value that x can take on
+     - BigInt
+   * - x.valueRange
+     - Return the range (x.minValue downto x.maxValue). Note can't be used for value which overflow the JVM Int capacity.
```

Comment:
- It is `to` not `downto`
- I would reformulate for those who don't know about JVM
- I am not sure if this use of "take on" is correct

What do you think about:

```
Return the range of values x can take. Bounds must not overflow a SInt(32 bits) (JVM restriction)
```

Reply:
fixed the direction, added a clearer description that does not elude to the JVM as much, removed the "take on" wording